src/components/Button/MucButton.vue

```diff
@@ -11,18 +11,21 @@
         v-if="icon"
         :icon="icon"
         class="m-button__icon"
+        :class="{ 'no-left-margin': !slots.default }"
```

Comment:
I think it would be better to not rely on custom classes here. Maybe a CSS utility library like https://github.com/tachyons-css/tachyons, https://github.com/basscss/basscss or https://github.com/turretcss/turretcss could be used?

Reply:
I think this would be overkill for what we are trying to accomplish here. If we write custom css like this it's in most cases to fix or extend the existing patternlab css.

Reply:
I don't now what patternlab css offers. Of course best solution would be if patternlab provides helper classes like m-0, p-0 and so on as well as other CSS libraries do it.

Reply:
A library would be too much - but i think we can create some custom classes like `ml-0`, `ml-1` ... in a centralized css-file. But this should be done in a separate issue

Reply:
Okay then please create a issue with helper css classes and do the migration later on. :)